averbis/core/_rest_client.py

```diff
@@ -659,6 +679,45 @@ def export_text_analysis(
             self.name, document_sources, process, annotation_types
         )
 
+    def list_pear_components(self) -> List[str]:
+        """
+        List all existing pear components.
+        :return: The pear component list.
+        """
+        # noinspection PyProtectedMember
+        return self.client._list_pear_components(self.name)
+
+    def delete_pear_component(self, pear_identifier: str) -> None:
+        """
+        Delete the pear component by identifier.
+        """
+        # noinspection PyProtectedMember
+        self.client._delete_pear_component(self.name, pear_identifier)
+        return None
+
+    def install_pear_component(self, file_or_path: Union[typing.IO, Path, str]) -> PearComponent:
+        """
+        Install a pear component by file or path.
+        """
+        if isinstance(file_or_path, str):
```

Comment:
The convenience methods should be slim. Better pull down the handling of different arguments types into the method in the Client class.

Reply:
k